dir.props

```diff
@@ -46,7 +46,7 @@
     <NugetRestoreCommand>$(NugetRestoreCommand) -OutputDirectory "$(PackagesDir.TrimEnd('/\'.ToCharArray()))"</NugetRestoreCommand>
     <NugetRestoreCommand>$(NugetRestoreCommand) $(NuGetConfigCommandLine)</NugetRestoreCommand>
     <NugetRestoreCommand>$(NugetRestoreCommand) -Verbosity detailed</NugetRestoreCommand>
-    <ResolveNugetPackages>false</ResolveNugetPackages>
```

Comment:
This is supposed to prevent project.json processing (it does). Now that I am deleting products.config, this is not needed (you were right, without it project.json and products.config conflict).